projects/gdal/build.sh

```diff
@@ -15,13 +15,28 @@
 #
 ################################################################################
 
+# build libcurl.a (builing against Ubuntu libcurl.a doesn't work easily)
+if [[ ! -d curl ]]; then
+    git clone --depth 1 https://github.com/curl/curl.git
```

Comment:
we recommend 'git clone' and such to be in the docker file -- this way we'll download the code once during the image creation and build multiple times (for different built configs) w/o downloading anything.